MapboxDirections/MBRouteStep.swift

```diff
@@ -785,6 +824,11 @@ open class RouteStep: NSObject, NSSecureCoding {
      */
     open let maneuverDirection: ManeuverDirection?
     
+    /**
+     Indicates what side of a bidirectional road the driver must be driving on. Also referred to as the rule of the road.
+     */
+    open let roadSide: RoadSide?
```

Comment:
This property is always returned, I think we can make this non-optional. Noting, it is in fact returned for biking/walking.

Reply:
Making it optional will also allow it to bridge to Objective-C.

Reply:
Sorry, I meant non-optional. It should be non-optional.